pyproject.toml

```diff
@@ -59,6 +59,7 @@ twine = ">=4.0.0,<6.0"
 tomlkit = ">=0.12.4,<1.0"
 lazy_loader = ">=0.4"
 reflex-chakra = ">=0.6.0"
+libsass = "0.23.0"
```

Comment:
Could we add this as an optional dependency so as not to increase the base dependency size? I'm also wondering if this would be better as a third-party package or if we should have it builtin to reflex, will let others give input there.

Reply:
I think it would be better to add to the list of optional dependencies and in “rx.App”, a configuration relating to sass/scss and when this configuration is valid, it requires “libsass”.